pkg/helmcatalog/actions/helm_actions.go

```diff
@@ -0,0 +1,13 @@
+package actions
+
+import "helm.sh/helm/v3/pkg/action"
+
+type HelmActions struct {
```

Comment:
Since `Action` remain uninitialized (see above), no need to wrap `Configuration` in another struct. Please remove.

Reply:
you mean not to have `HelmActions` struct ?

Reply:
yes.